microshift_networking/microshift-disconnected-network-config.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-disconnected-network-config"]
+= Configuring network settings for fully disconnected hosts
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-networking-disconnected-hosts
+toc::[]
+
+Learn how to apply networking customization and settings to run {microshift-short} on fully disconnected hosts.  A disconnected host should be the {op-system-base-full} operating system, versions 9.0+, whether real or virtual, that runs without network connectivity.
```

Comment:
```suggestion
Learn how to apply networking customization and settings to run {microshift-short} on fully disconnected hosts. A disconnected host should be the {op-system-base-full} operating system, versions 9.0+, whether real or virtual, that runs without network connectivity.
```

Reply:
@ShaunaDiaz I feel silly even mentioning this, but vale flagged it big-time. :-/

Reply:
Huh my local Vale in VSC didn't even twinge. I didn't see it 'cause of my line-wrap view. Good eye!